### PR TITLE
feat: added ASTR faucet

### PR DESCRIPTION
**Pull request by impelcrypto**
* Added ASTR faucet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,8 @@ FAUCET_SECRET_PHRASE=<secret phrase>
 
 # Faucet send amount
 TESTNET_FAUCET_AMOUNT = '15'
-MAINNET_FAUCET_AMOUNT = '0.002'
+SHIDEN_FAUCET_AMOUNT = '0.001'
+ASTAR_FAUCET_AMOUNT = '0.002'
```
Comment (hoonsubin) [inline README.md:29]: Any reason why Astar has more than Shiden? I thought it should be the other way around.

File: src/app.ts [hunk 2]
```diff
@@ -50,14 +49,15 @@ const discordFaucetApp = async (appCred: DiscordCredentials) => {
         throw new Error('No seed phrase was provided for the faucet account');
     }
 
+    const network = Network.shibuya;
```
Comment (hoonsubin) [inline src/app.ts:52]: Not clean to hard-code in a network connection on the entry point.

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (impelcrypto) [reply README.md:29]: Ok, I'll change it to 0.002.

**Review by hoonsubin — APPROVED** [review]
LGTM

**Review by hoonsubin — CHANGES_REQUESTED** [review]
Sorry, the previous review was wrong

File: src/app.ts (hunk 2 above)
Comment (impelcrypto) [reply src/app.ts]: Ok, fixed.

**Review by hoonsubin — APPROVED** [review]
LGTM

**Pull request merged**